README.md

```diff
@@ -1,7 +1,10 @@
 # Taxonomy
 
-[![Build Status](https://secure.travis-ci.org/silverstripe/silverstripe-taxonomy.png)](http://travis-ci.org/silverstripe/silverstripe-taxonomy)
-[![Code Coverage](https://codecov.io/gh/silverstripe/silverstripe-taxonomy/branch/master/graph/badge.svg)](https://codecov.io/gh/silverstripe/silverstripe-taxonomy)
+[![Build Status](https://img.shields.io/travis/silverstripe/silverstripe-taxonomy.svg)](http://travis-ci.org/silverstripe/silverstripe-taxonomy)
+[![Scrutinizer Code Quality](https://img.shields.io/scrutinizer/g/silverstripe/silverstripe-taxonomy.svg)](https://scrutinizer-ci.com/g/silverstripe/silverstripe-taxonomy/?branch=master)
+[![Code Coverage](https://img.shields.io/codecov/c/github/silverstripe/silverstripe-taxonomy.svg)](https://codecov.io/gh/silverstripe/silverstripe-taxonomy)
+[![Version](http://img.shields.io/packagist/v/silverstripe/taxonomy.svg?style=flat)](https://packagist.org/packages/silverstripe/silverstripe-sharedraftcontent)
```

Comment:
This is referencing sharedraftcontent. We only really need the Build Status, Code Quality and Code Coverage badges - you could remove the other two